libraries/rust/common/src/ibc/core/ics23_commitment/merkle.rs

```diff
@@ -67,193 +64,6 @@ impl From<MerkleProof> for RawMerkleProof {
     }
 }
 
-impl MerkleProof {
```

Comment:
Where would this be moved now?

Reply:
we dont need this anymore since we have our own merkle verification for btp messages.